builds/scala_service/run.sh

```diff
@@ -8,7 +8,9 @@ echo "Running project $PROJECT"
 
 echo "Fetching config from AWS..."
 aws s3 ls s3://$INFRA_BUCKET/$CONFIG_KEY
-aws s3 cp s3://$INFRA_BUCKET/$CONFIG_KEY /opt/docker/conf/application.ini
+aws s3 cp s3://$INFRA_BUCKET/$CONFIG_KEY /opt/docker/conf/application.ini.template
+
+envsubst < /opt/docker/conf/application.ini.template > /opt/docker/conf/application.ini
```

Comment:
Could we get a comment explaining exactly what this is meant to do? Double redirects like this are always fiddly.